doc/tutorial.md

```diff
@@ -135,13 +148,23 @@ The additional parameters are:
 
 ![Script execution with adjusted __chequeLimit__](assets/polkadot.js_execute_script_buy.png)
 
-The flow of this code example is:
-* Bob: `initial_coin_minting`, but only once after publication.
-* Alice: `register_new_user`, but only once after publication.
-* Alice: `buy_coin`.
-* Alice: `wash_car` for each time you bought a washing coin.
 
-You have now successfully published a Move module on your Polkadot blockchain and executed successfully Move scripts.
+## Finishing the Tutorial
+
+You have successfully published a Move module on your Polkadot blockchain and executed a Move script successfully. The past sections have explained how to:
+- [estimate the optimal amount of gas for publishing a module](#estimating-gas-for-module-publication),
+- [publish a module](#publication),
+- [create the script transaction](#creating-script-transactions),
+- [estimate the optimal amount of gas for executing a script](#estimating-gas-for-script-execution),
+- and how to [execute a script](#execution).
```

Comment:
Let's keep it short in order not to scare away people with a long tutorial page.
```suggestion
You have successfully published a Move module on your Polkadot blockchain and executed a Move script successfully.
```

Reply:
Done